app/javascript/components/mappers/domainToApi/lease.js

```diff
@@ -8,9 +7,9 @@ export const leaseFieldMapper = {
   'no_preference_used': 'no_preference_used',
   'preference_used': 'preference_used',
   'unit': 'unit',
-  total_monthly_rent_without_parking: (source) => currencyToFloat(source.total_monthly_rent_without_parking),
```

Comment:
same here re: flipping the order to avoid the lambda fn

Reply:
updated